src/components/head.tsx

```diff
@@ -0,0 +1,30 @@
+import { FC } from "react";
+
+type MetaData = {
+  title: string;
+  description: string;
+  icon: string;
+};
+
+const DEFAULT_META_DATA: MetaData = {
+  title: "React Tokyo",
+  description: "A React community in Tokyo",
+  icon: "/images/favicon.png",
+};
+
+/**
+ * ページのメタデータ設定用のコンポーネント
+ */
+export const Head: FC<Partial<MetaData>> = (metaProps) => {
```

Comment:
すみません、好みの範疇かもしれませんが、トレンドでもあると思うので、`FC`使わず、utility typesも使わず、これでいきたいです。

```suggestion
export const Head = (metaProps: Partial<MetaData>) => {
```

Reply:
@dai-shi 

ご確認ありがとうございます！
変更内容に異論ございませんので、ブランチに反映いたしました。

後学のため、2点ご質問させてください！

## ① 

> FC使わず

FCの利用について、
https://zenn.dev/motonosuke/articles/13d0fdd417ec7c#fc%E3%81%AE%E6%AC%A0%E7%82%B9%E3%81%AB%E3%81%A4%E3%81%84%E3%81%A6
こちらに記載されている欠点はあるかと認識しているのですが、
トレンドがFCを使わない方に傾いているのも上記の欠点が大きいでしょうか。

## ②

> utility typesも使わず、

こちらについて、今回の例だと「utility types = Partial」になりますでしょうか。
https://www.typescriptlang.org/docs/handbook/utility-types.html#partialtype

その場合、変更後のソースにもPartialはイキになっているかと思ってのご質問でした。

- - -

お手数ですが、どうぞよろしくお願いいたします！